src/Features/LanguageServer/Protocol/Handler/Hover/HoverHandler.cs

```diff
@@ -44,21 +45,15 @@ public HoverHandler(ILspSolutionProvider solutionProvider) : base(solutionProvid
             }
 
             var text = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
-            return new Hover
+
+            // TODO - Switch to markup content once it supports classifications.
+            // https://devdiv.visualstudio.com/DevDiv/_workitems/edit/918138
+            return new VSHover
             {
                 Range = ProtocolConversions.TextSpanToRange(info.Span, text),
-                Contents = new MarkupContent
-                {
-                    Kind = MarkupKind.Markdown,
-                    Value = GetMarkdownString(info)
-                }
+                Contents = new SumType<SumType<string, MarkedString>, SumType<string, MarkedString>[], MarkupContent>(string.Empty),
+                RawContent = await IntellisenseQuickInfoBuilder.BuildContentWithoutNavigationActionsAsync(info, document, cancellationToken).ConfigureAwait(false)
```

Comment:
maybe doc that we're not including this because lsp doesn't support hyperlinks?

(actually, is that hte case... seems surprising to me that this wouldn't be supported.

Reply:
Added comment.  Note that LSP does support hyperlinks in markupcontent, but the classified text elements use navigation actions for hyperlinks which are not serializable by the client.

So we basically have a choice between colors and links until support for colorized tooltips in LSP is added.